cranelift/codegen/src/ir/memflags.rs

```diff
@@ -265,6 +265,14 @@ impl MemFlags {
         self.set_vmctx();
         self
     }
+
+    pub(crate) fn as_u8(self) -> u8 {
+        self.bits
+    }
+
+    pub(crate) fn from_u8(bits: u8) -> Self {
+        Self { bits }
+    }
```

Comment:
These don't seem to be used. Could you remove them to avoid exposing that mem flags currently fit in a u8. They may not anymore in the future.